addons/website_blog/views/snippets/snippets.xml

```diff
@@ -28,5 +28,10 @@
     <xpath expr="//*[@data-js='anchor']" position="attributes">
         <attribute name="data-exclude" add=".o_wblog_post_content_field > :not(div, section)" separator=","/>
     </xpath>
+
+    <!-- Hides ContainerWidth option for content in blog posts -->
+    <xpath expr="//div[@data-js='ContainerWidth']" position="attributes">
+        <attribute name="data-exclude">#o_wblog_post_content *</attribute>
```

Comment:
@younn-o I am changing this to
```xml
<attribute name="data-exclude" add="#o_wblog_post_content *" separator=","/>
```
Otherwise you break potential custo by fully overriding the attribute